flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/TableSchema.java

```diff
@@ -346,6 +346,41 @@ public Schema toSchema() {
         return builder.build();
     }
 
+    /** Helps to migrate to the new {@link Schema} class, retain comments when needed. */
+    public Schema toSchema(Map<String, String> comments) {
```

Comment:
I like this method~

Reply:
please let `toSchema()` call `toSchema(Map<String, String> comments)`

Reply:
done :)